pkg/kubelet/kubelet.go

```diff
@@ -350,6 +352,31 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	if err != nil {
 		return nil, err
 	}
+	if len(thresholds) > 0 {
```

Comment:
@vishh Any reason container manager would be more appropriate than the eviction manager? Eviction manager seems better to me as it already knows whether eviction is enabled, and wrt https://github.com/kubernetes/kubernetes/issues/31676#issuecomment-244496413 `eviction_enabled && swap_enabled` is the necessary condition for an error.